torch/csrc/jit/mobile/import.cpp

```diff
@@ -171,11 +171,16 @@ std::unordered_set<std::string> BytecodeDeserializer::
   for (const auto& op : ops_list) {
     auto op_item = op.toTuple()->elements();
     TORCH_CHECK(
-        op_item.size() == 2, "There should be two parts in an operator name.");
+        op_item.size() >= 2,
```

Comment:
Maybe we should document here what parts are expected (I wish we have a better schema definition so we didn't have to but since we don't then we should describe it at least).


Also, in the future if we ever make the min version is >= that 6 we could remove this check, right? Just curious.

Reply:
Right. If we have the byte code version check >= 6, this check can be removed.